docs/data/material/migration/upgrade-to-v6/upgrade-to-v6.md

```diff
@@ -155,6 +155,13 @@ If the default heading element conflicts with your existing structure, you can c
 </Accordion>
 ```
 
+### Accordion (Starting from v6.3.0)
```

Comment:
I think it'd be better to merge this section with the previous one. Maybe something like this (with better copy? 😅):

```
### Accordion

#### Summary wrapped in a heading

#### Summary as a button (from v6.3.0)
```

Reply:
Makes sense. Done.